### DISCUSSION TITLE
Record both Audio and MIDI on one clip?

### DISCUSSION THREAD

**Discussion by Whiskeysaurus** [helgoboss/helgobox#1173] [Q&A]
Hello, I'm not quite sure how to do this, or if it's possible... For example, I'm using a Launchpad pro mk3, and I trigger recording of an external synth. How would I go about recording automation for a software effect at the same time? Obviously I could simultaneously record a clip in a separate column, but by I can't imagine the only way as that's not really functional live.

**Comment by helgoboss** [comment]
In the future, recording automation for a software effect will probably be possible via #1041.

Recording MIDI and audio together into one clip is not possible and there's also no existing feature request for that. I have been thinking about that possibility, but for other reasons (recording both VSTi audio output and MIDI, to have both).
